src/QZXing.pro

```diff
@@ -16,7 +16,7 @@
 
 include(QZXing.pri)
 
-VERSION = 2.3
+#VERSION = 2.3
```

Comment:
May i ask, why did you comment this out?

Reply:
Sorry!